lib/bambora/json_client.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
 module Bambora
-  class JSONRequest
-    attr_reader :client
+  class JSONClient < Bambora::Client
```

Comment:
Now, inherit from `Bambora::Client`. The difference here is the explicit `Content-Type` header below, and the JSON specific parsing methods.